src/services/appInsightsService.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line no-undef
+export default appInsights
```

Comment:
The appInsights instance is likely accessible in this scope as window.appInsights. That said, it's probably best to merge this into src/appInsights.js or to merge that file into this.